tests/slr_integration/Admin/Maps_Layout_Homepage_Test.php

```diff
@@ -23,16 +30,28 @@ public function test_empty_seating_configurations(): void {
 	}
 
 	public function test_empty_layouts(): void {
-		// Become administrator.
-		wp_set_current_user( self::factory()->user->create( [ 'role' => 'administrator' ] ) );
-		// Simulate a request to the Seat Configurations and Layouts home page.
-		$_GET['page'] = 'tec-events-assigned-seating';
-		$_GET['tab']  = Layouts::get_id();
-
 		ob_start();
 		tribe( Maps_Layouts_Home_Page::class )->render();
 		$html = ob_get_clean();
 
 		$this->assertMatchesHtmlSnapshot( $html );
 	}
-}
\ No newline at end of file
+	
+	public function test_maps_tab_card_listing() {
+		$this->set_class_fn_return(
```

Comment:
Let's avoid mocking where possible, use `Tables\Map::insert_many` to insert mock data in the real table.
This will cover more ground in the integration test.
Use a `@before` and `@after` method to truncate the maps custom table before each test.